### PR TITLE
fix(config): add interval option for git configuration

### LINKED ISSUE #46: Make git fetch interval configurable
See: https://github.com/get-glu/glu/pull/45

Glu now fetches periodically (asynchronously from calls to `View`).
We should make this interval configurable on the remote here:
https://github.com/get-glu/glu/blob/e88e710e168b26cd064ea3a1a0fb5892982dc2be/pkg/config/git.go#L61-L65

### PR DESCRIPTION
**Pull request by erka**
closes #46

### CODE REVIEW

File: pkg/config/git.go [hunk 1]
```diff
@@ -53,15 +56,19 @@ func (r *GitRepository) setDefaults() error {
 
 			remote.Name = "origin"
 		}
+		if remote.Interval == 0 {
```
Comment (markphelps) [inline pkg/config/git.go:59]:
```suggestion
		if remote.Interval < 1 {
```

suggestion: since duration can be negative

**Review by markphelps — COMMENTED** [review]
one minor nit. ty!

**Review by markphelps — APPROVED** [review]
ty!!

**Comment by GeorgeMac** [comment]
Legend!

**Pull request merged**